src/java/install.sh

```diff
@@ -195,28 +196,33 @@ check_packages() {
     esac
 }
 
-# Use Microsoft JDK for everything but JDK 8 and 18 (unless specified differently with jdkDistro option)
-get_jdk_distro() {
-    VERSION="$1"
-    if [ "${JDK_DISTRO}" = "ms" ]; then
-        if echo "${VERSION}" | grep -E '^8([\s\.]|$)' > /dev/null 2>&1 || echo "${VERSION}" | grep -E '^18([\s\.]|$)' > /dev/null 2>&1; then
-            JDK_DISTRO="tem"
-        fi
-    fi
-}
-
 find_version_list() {
     prefix="$1"
     suffix="$2"
     install_type=$3
     ifLts="$4"
     version_list=$5
+    java_ver=$6
+    
+    check_packages jq
+    all_lts_versions=$(curl -s https://api.adoptium.net/v3/info/available_releases)
     if [ "${ifLts}" = "true" ]; then 
-        all_lts_versions=$(curl -s https://api.adoptium.net/v3/info/available_releases)
         major_version=$(echo "$all_lts_versions" | jq -r '.most_recent_lts')
-        regex="${prefix}\\K${major_version}\\.?[0-9]*\\.?[0-9]*${suffix}"
-    else 
+    elif [ "${java_ver}" = "latest" ]; then
+        major_version=$(echo "$all_lts_versions" | jq -r '.most_recent_feature_release') 
```

Comment:
How is this latest related to LTS? 

Java could release 22 as latest and not make it LTS for a while, then this would fail, right?

Reply:
@prathameshzarkar9 Can you help respond to this comment?